hurricane/server/__init__.py

```diff
@@ -153,3 +155,45 @@ def callback_command_exception_check(future: asyncio.Future, webhook_url: str =
             current_loop.stop()
         else:
             logger.info("Execution of management commands was successful. Webhook is not set")
+
+
+def check_databases():
+    for db_name in connections:
+        connection = connections[db_name]
+        cursor = connection.cursor()
+        try:
+            cursor.execute("SELECT (1)")
+            logger.info("Database was checked successfully")
+            return True
+        except Exception as e:
+            logger.warning(f"Database command execution has failed with {e}")
+            return False
+        finally:
```

Comment:
That is an dangerous Python construct. The `finally` block get's executed even though you already `return`'ed (ref https://docs.python.org/3/tutorial/errors.html#defining-clean-up-actions). It might be ok in this case but please be aware. In my opinion "return-finally" is a bad pattern.